src/runtime-rs/crates/runtimes/virt_container/src/container_manager/manager.rs

```diff
@@ -192,6 +210,22 @@ impl ContainerManager for VirtContainerManager {
             .get(container_id)
             .ok_or_else(|| Error::ContainerNotFound(container_id.clone()))?;
         c.start(process).await.context("start")?;
+
+        // Poststart Hooks:
+        // * should be run in runtime namespace
```

Comment:
IIUC, runtime namespace means container namespace, do you need to enter the network namespace?  like `netns::NetnsGuard::new` Do I missing something?

Reply:
According to the spec, `runtime namespace` is not `container namespace`.